src/shared/components/query/CancerStudyTreeData.ts

```diff
@@ -252,3 +264,12 @@ export default class CancerStudyTreeData {
         }
     }
 }
+
+function flattenTags(tags: any): string {

```

Comment:
stringify tags?

Reply:
Renamed in [2009473](https://github.com/cBioPortal/cbioportal-frontend/pull/4571/commits/2009473b6d81c094ba4246ee45947fd9d316e4fe)